audtorch/transforms/transforms.py

```diff
@@ -790,17 +830,20 @@ class Spectrogram(object):
 
     """
 
-    def __init__(self, window_size, hop_size, *, window='hann', axis=-1):
+    def __init__(self, window_size, hop_size, *, n_fft=None,
```

Comment:
As we use `window_size` and `hop_size` at the moment, a better name might be `fft_size`?

Reply:
Agreed.